ring-devel/src/ring/middleware/stacktrace.clj

```diff
@@ -13,6 +13,7 @@
 
 (defn wrap-stacktrace-log
   "Wrap a handler such that exceptions are logged to *err* and then rethrown.
+
```

Comment:
What's the purpose of this newline?

Reply:
It matches the formatting on line 112 of the same file, and line 25 of reload.clj in the same directory.